Link failure in spicy-build when using `&convert`

With a build yesterday, my first new one that I have done since 20-Mar-2020, now running the resulting spicy-config / spicyc against something other than HelloWorld.spicy smoke-tests, spicy-build is reporting:
```
spicy/build/bin/spicy-build -o J_S.exe JSON_SCCP.spicy
Undefined symbols for architecture x86_64:
  "_jrx_current_accept", referenced from:
      hilti::rt::regexp::MatchState::_advance(hilti::rt::stream::View const&, bool) in libhilti-rt.a(regexp.cc.o)
  "_jrx_is_anchored", referenced from:
      hilti::rt::RegExp::_search_pattern(jrx_match_state*, hilti::rt::Bytes const&, int*, int*, bool, bool) const in libhilti-rt.a(regexp.cc.o)
  "_jrx_match_state_copy", referenced from:
      hilti::rt::regexp::MatchState::MatchState(hilti::rt::regexp::MatchState const&) in libhilti-rt.a(regexp.cc.o)
      hilti::rt::regexp::MatchState::operator=(hilti::rt::regexp::MatchState const&) in libhilti-rt.a(regexp.cc.o)
  "_jrx_match_state_done", referenced from:
      hilti::rt::regexp::MatchState::operator=(hilti::rt::regexp::MatchState const&) in libhilti-rt.a(regexp.cc.o)
      hilti::rt::regexp::MatchState::~MatchState() in libhilti-rt.a(regexp.cc.o)
      hilti::rt::regexp::MatchState::~MatchState() in libhilti-rt.a(regexp.cc.o)
      hilti::rt::RegExp::find(hilti::rt::Bytes const&) const in libhilti-rt.a(regexp.cc.o)
      hilti::rt::RegExp::_search_pattern(jrx_match_state*, hilti::rt::Bytes const&, int*, int*, bool, bool) const in libhilti-rt.a(regexp.cc.o)
      hilti::rt::RegExp::findSpan(hilti::rt::Bytes const&) const in libhilti-rt.a(regexp.cc.o)
      hilti::rt::RegExp::findGroups(hilti::rt::Bytes const&) const in libhilti-rt.a(regexp.cc.o)
      ...
  "_jrx_match_state_init", referenced from:
      hilti::rt::regexp::MatchState::MatchState(hilti::rt::RegExp const&) in libhilti-rt.a(regexp.cc.o)
      hilti::rt::RegExp::_search_pattern(jrx_match_state*, hilti::rt::Bytes const&, int*, int*, bool, bool) const in libhilti-rt.a(regexp.cc.o)
  "_jrx_num_groups", referenced from:
      hilti::rt::RegExp::findGroups(hilti::rt::Bytes const&) const in libhilti-rt.a(regexp.cc.o)
  "_jrx_regerror", referenced from:
      hilti::rt::RegExp::_compileOne(std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> >, int) in libhilti-rt.a(regexp.cc.o)
  "_jrx_regexec_partial", referenced from:
      hilti::rt::regexp::MatchState::_advance(hilti::rt::stream::View const&, bool) in libhilti-rt.a(regexp.cc.o)
      hilti::rt::RegExp::_search_pattern(jrx_match_state*, hilti::rt::Bytes const&, int*, int*, bool, bool) const in libhilti-rt.a(regexp.cc.o)
  "_jrx_regfree", referenced from:
      hilti::rt::RegExp::_newJrx() in libhilti-rt.a(regexp.cc.o)
      std::__1::__shared_ptr_pointer<jrx_regex_t*, hilti::rt::RegExp::_newJrx()::$_0, std::__1::allocator<jrx_regex_t> >::__on_zero_shared() in libhilti-rt.a(regexp.cc.o)
  "_jrx_reggroups", referenced from:
      hilti::rt::RegExp::_search_pattern(jrx_match_state*, hilti::rt::Bytes const&, int*, int*, bool, bool) const in libhilti-rt.a(regexp.cc.o)
      hilti::rt::RegExp::findGroups(hilti::rt::Bytes const&) const in libhilti-rt.a(regexp.cc.o)
  "_jrx_regset_add", referenced from:
      hilti::rt::RegExp::_compileOne(std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> >, int) in libhilti-rt.a(regexp.cc.o)
  "_jrx_regset_finalize", referenced from:
      hilti::rt::RegExp::RegExp(std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> >, hilti::rt::regexp::Flags) in libhilti-rt.a(regexp.cc.o)
      hilti::rt::RegExp::RegExp(std::__1::vector<std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> >, std::__1::allocator<std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> > > > const&, hilti::rt::regexp::Flags) in libhilti-rt.a(regexp.cc.o)
  "_jrx_regset_init", referenced from:
      hilti::rt::RegExp::_newJrx() in libhilti-rt.a(regexp.cc.o)
ld: symbol(s) not found for architecture x86_64
clang: error: linker command failed with exit code 1 (use -v to see invocation)
```
The common denominator is many start with ` _jrx_`, and many reference  
      `hilti::rt::regexp::`

Does that suggest what is missing in my configuration, as the cause?

--- a/hilti/CMakeLists.txt
+++ b/hilti/CMakeLists.txt
@@ -183,6 +183,12 @@ foreach ( lib hilti-rt hilti-rt-debug )
 
     add_library(${lib} STATIC)
     target_link_libraries(${lib} ${lib}-objects)
+
+    # Since only an OBJECT libraries usage requirements are propagated
+    # transitively, but not its object files, explicitly link against
+    # jrx-objects which is an OBJECT library itself and depended on by
+    # `jrx-objects`.
+    target_link_libraries(${lib} jrx-objects)
 endforeach ()
 
 # Build hilti-rt with release flags.

--- a/tests/spicy/tools/spicy-build-jrx.spicy
+++ b/tests/spicy/tools/spicy-build-jrx.spicy
@@ -1,0 +1,13 @@
+# @TEST-GROUP: no-jit
+# @TEST-EXEC: ${SPICY_BUILD} %INPUT
+
+module Foo;
+
+public type LengthType = unit {
+  islong: uint8;
+
+  switch ( self.islong ) {
+    # Include a `&convert` here which triggers a dependency on justrx.
+    1 -> length_parse: bytes &size=self.islong &convert=$$.to_uint();
+  };
+};

